test/QueryPlannerTestHelpers.h

```diff
@@ -245,6 +246,12 @@ inline auto TransitivePath =
                             TransitivePathSideMatcher(right))));
     };
 
+inline auto Sort = [](const std::same_as<QetMatcher> auto&... childMatchers) {
+  return RootOperation<::Sort>(
+      AllOf(Property("getChildren", &Operation::getChildren,
+                     ElementsAre(Pointee(childMatchers)...))));
+};
+
```

Comment:
I think you can use the shorter `MatchTypeAndOrderedChildren` shorthand to get rid of code duplication.
And add a short comment.